parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -210,4 +211,16 @@ systemctl_restart() {
             sleep $wait_sleep
         fi
     done
+}
+sysctl_reload() {
+    retries=$1; wait_sleep=$2; timeout=$3
+    for i in $(seq 1 $retries); do
+        timeout $timeout sysctl --system
+        [ $? -eq 0  ] && break || \
```

Comment:
There's an extra space after `0`. Also I think this would be simpler as:
```shell
timeout $timeout sysctl --system && break || \
```

Reply:
This is copy/pasta, let's use your suggested fix in a follow-up PR to all funcs in this helpers file

Reply:
Agreed—let me know whenever the CIS dust has settled enough, because I have lots of script refactoring and linting ready to unleash.